use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+# Init process for Fluent Bit on ECS, multi-config support
+
+
+
+### What is init process and multi-config support?
+
+Multi-config support means when you use a Fluent Bit image with `init` tag on ECS, you can use some config files conveniently to configure Fluent Bit. These config files can be your own files that you upload to S3, or you can use the [built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs) we provide directly. Init process achieves this feature. 
+
+
+
+### Why we create init process?
+
+Currently, if you want to set up additional configs for Fluent Bit running on ECS, you need to [build a custom image](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file). And you cannot set up a [PARSER] section for Fluent Bit because using a parser in Fluent Bit requires you to change the Fluent Bit command or specify the path of the parser file under the [SERVICE] section.
```

Comment:
> And you cannot set up a [PARSER] section for Fluent Bit

And you can not easily set up... 


They can set it up before init... its just not convenient

Reply:
Also let's add a sentence at the end like "With the init process for Fluent Bit, these import statements are not needed, init will automatically add them."

Reply:
Yeah.. will change the sentence here

Reply:
> requires you to change the Fluent Bit command

I'd prefer to remove this actually since almost no users use the command option. Few know it exists. Eveyone uses the import in the [SERVICE], so let's focus on that.